tests/migration_tool/tests_migrate.py

```diff
@@ -31,7 +36,7 @@ def setUp(self):
         """Set up the utils tests."""
         super().setUp()
         public_tenant = Tenant.objects.get(tenant_name="public")
-        Group.objects.create(name="default", tenant=public_tenant, platform_default=True)
+        default_group = Group.objects.create(name="default", tenant=public_tenant, platform_default=True, system=True)
```

Comment:
I wonder if we can use `seed_group()` instead in the setup (probably after the roles are created then)? This would ensure test setup stays consistent with the app and maybe reduce some of this code.

Reply:
I added `seed_group` and `clone_default_group_in_public_schema`

it is possible use `seed_group()`, function creates also admin default group